project-koku/koku
# Meta cleanup

Fixes [COST-2254](https://issues.redhat.com/browse/COST-2254)

Changes proposed in this PR:
* Cleaning up code that use to return the users preferred setting in meta data
*

Testing Instructions:
1. pull branch
2. bring up docker
2. create and load test data
3. Expect to see this result:
```
   GET http://127.0.0.1:8000/api/cost-management/v1/cost-type/

  
{
    "meta": {
        "count": 3
    },
    "links": {
        "first": "/api/cost-management/v1/cost-type/?limit=3&offset=0",
        "next": null,
        "previous": null,
        "last": "/api/cost-management/v1/cost-type/?limit=3&offset=0"
    },
    "data": [
        {
            "code": "unblended_cost",
            "name": "Unblended",
            "description": "Usage cost on the day you are charged"
        },
        {
            "code": "savingsplan_effective_cost",
            "name": "Amortized",
            "description": "recurring and/or upfront costs are distributed evenly across the month"
        },
        {
            "code": "blended_cost",
            "name": "Blended",
            "description": "Using a blended rate to calculate cost usage"
        }
    ]
}
```
4. there was previously a preferred cost-type in the meta with count so insure that is not there


---
Additional Context:
